core/app/helpers/spree/products_helper.rb

```diff
@@ -35,6 +35,11 @@ def default_variant(variants, product)
       variants_option_types_presenter(variants, product).default_variant || product.default_variant
     end
 
+    def should_display_compare_at_price(default_variant)
```

Comment:
if this method return `true/false` we could change name to: `should_display_compare_at_price?(default_variant)` wdyt ?